### PR TITLE
Add welcome page and update the title

### PR DESCRIPTION
**Pull request by audgirka**
This PR 

- Moves the welcome page under it's own folder `features/welcomePage`,
  related webview part under `webviews/apps/welcomePage`,
  copied exiting css under `media/welcomePage/style.css`.

- Updates the `extension.ts` and `webpack.config.ts` file to use the new welcome page.

- Adds initial ui tests for the welcome page

Fixes: [AAP-27791](https://issues.redhat.com/browse/AAP-27791)

### CODE REVIEW

File: test/ui-test/welcomePageUITest.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  By,
+  SideBarView,
+  ViewControl,
+  ViewSection,
+  WebView,
+} from "vscode-extension-tester";
+import { sleep } from "./uiTestHelper";
+
+config.truncateThreshold = 0;
+export function WelcomePageUITest(): void {
```
Comment (alisonlhart) [inline test/ui-test/welcomePageUITest.ts:13]:
```suggestion
export function welcomePageUITest(): void {
```
Changed to match case of other tests

**Review by alisonlhart — COMMENTED** [review]
Left one minor comment, otherwise looks good outside of the codecov report

File: test/ui-test/welcomePageUITest.ts (hunk 1 above)
Comment (audgirka) [reply test/ui-test/welcomePageUITest.ts]: updated name casing and related imports

**Comment by audgirka** [comment]
@TamiTakamiya can you help figure out why a few of lightspeed trial tests are failing

**Comment by TamiTakamiya** [comment]
@audgirka I created a branch based on yours and could recreate the problem in CI, i.e. the UI test failed on the One Click Trial test case.  Several findings in my tests are:

1. The test case runs successfully in the local environment
2.  Even in the CI environment, the test case runs w/o problem if it is executed by itself (w/o running others)
3.  If the UI test task is split into two parts (1) One Click Trial test, and (2) Others, both completes successfully in the CI environment, but code coverrage reports are not generated properly because two separate reports are created. Additional coding would be needed for merging two reports into one.

Though it is not clear why the test case fails in the CI environment, 3 (=running the One Click Trial test case separately) would be a good workaround and I would like to take that approach with a separate pull request.  

For this PR, I think we can temporarily disable the One Click Trial test by adding [this change](https://github.com/ansible/vscode-ansible/pull/1529/files#diff-cd2d359855d0301ce190f1ec3b4c572ea690c83747f6df61c9340720e3d2425e). Would you try it? Thanks.

**Comment by alisonlhart** [comment]
@audgirka There are some missing features from what's shown in the wireframe. Is it still planned to add a "Walkthroughs" section as shown in the mockup?

File: src/features/welcomePage/index.ts [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+                    <a href="https://docs.ansible.com/ansible/latest/getting_started/index.html">
+                      Learn Ansible development
+                      <span class="codicon codicon-link-external"></span>
+                    </a>
+                  </h3>
+                  <p>End to end course that will help you master automation development.</p>
+                </div>
+                <div class="catalogue">
+                  <h3>Once you are in the YAML file:</h3>
+                  <p>click Ctrl+L to fire the Ansible Lightspeed AI assistance for editing and explaining code.</p>
```
Comment (alisonlhart) [inline src/features/welcomePage/index.ts:170]: Should this note be included on this page or inside one of the walkthroughs (or omitted?)

**Review by alisonlhart — APPROVED** [review]

File: src/features/welcomePage/index.ts (hunk 2 above)
Comment (audgirka) [reply src/features/welcomePage/index.ts:170]: This is something that I just copied over from the other file, no cleanup done as it was not related to the changes for this PR.

**Pull request merged**